react/features/toolbox/components/native/HangupMenu.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+
+import React, { useCallback } from 'react';
+import { TouchableWithoutFeedback, View } from 'react-native';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { createBreakoutRoomsEvent, createToolbarEvent, sendAnalytics } from '../../../analytics';
+import { appNavigate } from '../../../app/actions';
+import { ColorSchemeRegistry } from '../../../base/color-scheme';
+import { endConference } from '../../../base/conference';
+import { BottomSheet, hideDialog } from '../../../base/dialog';
+import { getLocalParticipant, PARTICIPANT_ROLE } from '../../../base/participants';
+import Button from '../../../base/react/components/native/Button';
+import { BUTTON_TYPES } from '../../../base/react/constants';
+import { moveToRoom } from '../../../breakout-rooms/actions';
+import { isInBreakoutRoom } from '../../../breakout-rooms/functions';
+
+/**
+ * Component that renders the hangup menu.
+ *
+ * @returns {React$Element<any>}
+ */
+function HangupMenu() {
+    const dispatch = useDispatch();
+    const _styles = useSelector(state => ColorSchemeRegistry.get(state, 'Toolbox'));
+    const inBreakoutRoom = useSelector(isInBreakoutRoom);
+    const isModerator = useSelector(state => getLocalParticipant(state).role === PARTICIPANT_ROLE.MODERATOR);
+    const { DESTRUCTIVE, SECONDARY } = BUTTON_TYPES;
+
+    const handleCancel = useCallback(() => {
+        dispatch(hideDialog());
+    }, [ hideDialog ]);
+
+    const handleEndConference = useCallback(() => {
+        dispatch(hideDialog());
+        dispatch(endConference());
+    }, [ hideDialog ]);
+
+    const handleLeaveConference = useCallback(() => {
+        dispatch(hideDialog());
+        sendAnalytics(createToolbarEvent('hangup'));
+        dispatch(appNavigate(undefined));
+    }, [ hideDialog ]);
+
+    const handleLeaveBreakoutRoom = useCallback(() => {
+        dispatch(hideDialog());
+        sendAnalytics(createBreakoutRoomsEvent('leave'));
+        dispatch(moveToRoom());
+    }, [ hideDialog ]);
+
+    return (
+        <BottomSheet
+            onCancel = { handleCancel }>
+            <TouchableWithoutFeedback
```

Comment:
Why do you need this container? IIRC the bottom sheet implementation should take care of this.

Reply:
Fixed.